Makefile

```diff
@@ -77,6 +78,10 @@ pip-compile:
 docker:
 	git archive --format=tar.gz HEAD | docker build -t hypothesis/lms:$(DOCKER_TAG) -
 
+.PHONY: docker
+docker-dev:
```

Comment:
This points to the local code as opposed to pointing to git HEAD. Pointing at git HEAD would require the code to be committed before testing which of course is a pain when developing code locally so here we don't require that.